SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -32,6 +32,37 @@ extension Array {
     }
 }
 
+extension Array {
+    /// Separates this array's elements into chunks with a given length.
+    ///
+    /// If `length` is not a positive number or if it's greater than or equal to this array's length, we return a new array
+    /// whose sole element is this array itself.
+    ///
+    /// - parameter length: Maximum number of elements in each chunk.
+    /// - parameter includeRemainder: If set to `true`, include the last chunk in returned array even if its length is less than
+    ///   `length`. If set to `false`, discard the last chunk if its length is less than `length`. Defaults to `true`.
+    ///
+    /// - returns: New array of chunks of elements in this array.
+    public func ip_chunks(ofLength length: Int, includeRemainder: Bool = true) -> [[Element]] {
```

Comment:
I think a signature of `ip_chunks(of length: Int, includeRemainder: Bool = true) ...`